dist/vespa.spec

```diff
@@ -6,6 +6,10 @@
 # Only strip debug info
 %global _find_debuginfo_opts -g
 
+# Go binaries' build-ids are not recognized by RPMs yet, see
+# https://github.com/rpm-software-management/rpm/issues/367.
+%global _missing_build_ids_terminate_build 0
```

Comment:
This ignores build-ids for all packages.  I actually only need it for the clients subpackage, but don't know if, say, putting the %global statement in `%package clients` would work.